packages/plugin-node/src/index.ts

```diff
@@ -428,10 +428,8 @@ export async function buildEntrypoint({
   installedPaths?: Set<string>;
 }) {
   // Unique hash that will be used as directory name for `.output`.
-  const entrypointHash = crypto
-    .createHash('sha256')
-    .update(entrypoint)
-    .digest('hex');
+  // We're using the same format as for our other CLI Plugins.
+  const entrypointHash = getInputHash(`api/**/*.js`);
```

Comment:
This means we're using a single directory for all files that `vercel-plugin-node` handles.

I don't see an issue with that.